### PR TITLE
Design system

### PR DESCRIPTION
**Pull request by samuel-gomez-axa**
## Related issue

redesign CSS / HTML documentation in complete Design System 

### Person(s) for reviewing proposed changes

@guillaumechervet

### CODE REVIEW

File: storybook/design-system/package.json [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    "start": "gulp dev",
+    "build": "gulp build",
+    "deploy": "npm run build && serve -s dist",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint": "run-s lint:**",
+    "lint:js": "eslint ./src --fix",
+    "lint:pug": "pug-lint ./src",
+    "lint:scss": "stylelint \"src/**/*.scss\" --formatter \"verbose\" --max-warnings 96 --fix"
+  },
+  "author": "Samuel Gomez",
```
[the remark this answers is not in the document]
Comment (samuel-gomez-axa) [reply storybook/design-system/package.json]: done

File: storybook/design-system/package.json [hunk 2]
```diff
@@ -0,0 +1,116 @@
+{
+  "name": "design-system",
+  "version": "1.0.0",
+  "description": "",
```
[the remark this answers is not in the document]
Comment (samuel-gomez-axa) [reply storybook/design-system/package.json]: done

File: storybook/design-system/package.json [hunk 3]
```diff
@@ -0,0 +1,116 @@
+{
+  "name": "design-system",
+  "version": "1.0.0",
```
[the remark this answers is not in the document]
Comment (samuel-gomez-axa) [reply storybook/design-system/package.json]: done

File: storybook/design-system/scripts/publish.js [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+try {
+
+    const dir = './tmp';
+    if (!fs.existsSync(dir)){
+        fs.mkdirSync(dir);
+    }
+
+    const execSync = require('child_process').execSync;
+    let child = execSync(`git clone https://toto:${GITHUB_TOKEN}@github.com/AxaGuilDEv/AxaGuilDEv.github.io ./tmp/AxaGuilDEv.github.io/`);
+    console.log('error', child.error);
```
[the remark this answers is not in the document]
Comment (samuel-gomez-axa) [reply storybook/design-system/scripts/publish.js]: Sorry i forget to delete, done

File: storybook/design-system/package.json [hunk 5]
```diff
@@ -0,0 +1,116 @@
+{
+  "name": "design-system",
```
[the remark this answers is not in the document]
Comment (samuel-gomez-axa) [reply storybook/design-system/package.json]: done

File: storybook/design-system/package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (guillaumechervetaxa) [reply storybook/design-system/package.json]: Top, il ne me reste plus qua l'éxécuter sur mon poste, j'ai du tout recloner

**Pull request merged**